src/routes/user/register/1.svelte

```diff
@@ -0,0 +1,384 @@
+
+	const register_languages = async () => {
+		const zxcvbnCommonPackage = await import('@zxcvbn-ts/language-common');
+		const zxcvbnEnPackage = await import('@zxcvbn-ts/language-en');
+		const zxcvbnCsPackage = await import('@zxcvbn-ts/language-cs');
+		const zxcvbnDePackage = await import('@zxcvbn-ts/language-de');
+		const zxcvbnEsEsPackage = await import('@zxcvbn-ts/language-es-es');
+		const zxcvbnFiPackage = await import('@zxcvbn-ts/language-fi');
+		const zxcvbnFrPackage = await import('@zxcvbn-ts/language-fr');
+		const zxcvbnIdPackage = await import('@zxcvbn-ts/language-id');
+		const zxcvbnItPackage = await import('@zxcvbn-ts/language-it');
+		const zxcvbnJaPackage = await import('@zxcvbn-ts/language-ja');
+		const zxcvbnNlBePackage = await import('@zxcvbn-ts/language-nl-be');
+		const zxcvbnPlPackage = await import('@zxcvbn-ts/language-pl');
+		const zxcvbnPtBrPackage = await import('@zxcvbn-ts/language-pt-br');
+
+		const options: OptionsType = {
+			translations: zxcvbnEnPackage.translations,
+			graphs: zxcvbnCommonPackage.adjacencyGraphs,
+			dictionary: {
+				...zxcvbnCommonPackage.dictionary,
+				...zxcvbnCsPackage.dictionary,
+				...zxcvbnDePackage.dictionary,
+				...zxcvbnEsEsPackage.dictionary,
+				...zxcvbnFiPackage.dictionary,
+				...zxcvbnFrPackage.dictionary,
+				...zxcvbnIdPackage.dictionary,
+				...zxcvbnItPackage.dictionary,
+				...zxcvbnJaPackage.dictionary,
+				...zxcvbnNlBePackage.dictionary,
+				...zxcvbnPlPackage.dictionary,
+				...zxcvbnPtBrPackage.dictionary,
+				...zxcvbnEnPackage.dictionary,
+
+				// Check inputs
+				userInputs: [password?.value ?? '', email?.value ?? '', confirm_password?.value ?? '']
+			},
+			useLevenshteinDistance: true
+		};
+
+		zxcvbnOptions.setOptions(options);
+	};
+
+	$effect.pre(() => {
+		// register_languages();
+	});
+
+	// bind:value needs to be defined first to bind
+	// otherwise shows: typeerror $.get(...) is undefined
+	let email = $state<FormFieldState>({ value: "", error: new Array<string>() }),
+		password = $state<FormFieldState>({ value: "", error: new Array<string>() }),
+		confirm_password = $state<FormFieldState>({ value: "", error: new Array<string>() });
+
+	$effect(() => {
+		// update only if state exists on combined_state
+		if (combined_state.email) {
+			email = {
+				value: combined_state?.email ?? "",
+				error: new Array<string>()
+			};
+		};
+		if (combined_state.password) {
+			password = {
+				value: combined_state?.password ?? "",
+				error: new Array<string>()
+			};
+		};
+		if (combined_state.confirm_password) {
+			confirm_password = {
+				value: combined_state?.confirm_password ?? "",
+				error: new Array<string>()
+			};
+		};
+	});
+
+	let password_strength = $state(0);
+	let form_is_submitable = $state(false);
+
+	function check_if_form_is_submitable() {
+		form_is_submitable = [email, password, confirm_password].every((field) => {
+			return field?.value && _.isEmpty(field.error);
```

Comment:
remove the use of lodash

Reply:
Resolved on https://github.com/baseplate-admin/coreproject-v4-ui/pull/35/commits/8799b1022ea189c01b29c3f6d49b60c1fb5e05e9